circuit-types/src/keychain.rs

```diff
@@ -257,6 +257,12 @@ impl PublicSigningKey {
         let verifying_key = K256VerifyingKey::from(self);
         verifying_key.to_sec1_bytes().to_vec()
     }
+
+    /// Convert the key to bytes in uncompressed form
```

Comment:
We can probably delete the compressed bytes method now that we've switched serialization